src/app/meetups/view-meetups/meetups-view.component.ts

```diff
@@ -174,4 +174,11 @@ export class MeetupsViewComponent implements OnInit, OnDestroy {
     }
   }
 
+  routeToAssignee(assignedTo: string) {
```

Comment:
It looks like this is the same as the `routeToCreator` function on this component.  Instead of copying the function you can just use the other one.

For better naming let's call it `routeToUser` and call the variable `username`.